src/arrayMethodJoin.js

```diff
@@ -5,7 +5,26 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
you can add a default value for separator
```suggestion
  [].__proto__.join2 = function(separator = ',') {
```